eventmesh-runtime/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     implementation project(":eventmesh-webhook:eventmesh-webhook-api")
     implementation project(":eventmesh-webhook:eventmesh-webhook-receive")
 
+    implementation project(":eventmesh-retry:eventmesh-retry-api")
+    implementation project(":eventmesh-retry:eventmesh-retry-rocketmq")
```

Comment:
I don't think this dependency should be removed. If it is removed, the SPI will not take effect when the user specifies storage = rocketmq.

---

我觉得不应该删除此依赖, 如果删除, 当用户指定了storage=rocketmq的时候, 将不会生效此SPI.

Reply:
I think you are right. I removed the dependency just now.

Reply:
I rollbacked the dependency changes, and then i will resubmit it after we are done discussing it.

Reply:
The core question may be whether the user needs to declare the retry-rocketmq dependency.

Reply:
> I sincerely believe that there is still a need for another reviewer to express their views on this. @xwm1992. Because perhaps I misunderstood something. Thank you for your timely response.
> 
> 我真诚地觉得还是需要有另一位reviewer来就此说一下看法，@xwm1992 ，因为也许是我哪里理解得不对。感谢您及时的回应。

In fact, if the runtime does not show the addition of dependencies, it means that users need to load the dependencies of this SPI implementation by themselves. I am worried that it will create a greater burden for users.

---

其实如果runtime不显示加入依赖, 就代表了用户需要自行去加载这个SPI实现的依赖, 我担心对于用户会产生更大的负担,

Reply:
> I sincerely believe that there is still a need for another reviewer to express their views on this. @xwm1992. Because perhaps I misunderstood something. Thank you for your timely response.
> 
> 我真诚地觉得还是需要有另一位reviewer来就此说一下看法，@xwm1992 ，因为也许是我哪里理解得不对。感谢您及时的回应。

And because the runtime is generally an independent server, users can only make this switch take effect through a plugin, which is obviously more troublesome.

---

并且由于runtime一般是一个独立的服务器, 这样用户又只能通过如plugin的方式才能让此开关生效, 显然比较麻烦.